install.sh

```diff
@@ -114,20 +115,53 @@ EOF
                 echo "[i] File created."
 
                 # add cronjob
-                read -r -p "[?] Create a cronjob/systemd-service which updates /etc/hosts with new adservers once a week? [systemd/cronjob/N] " REPLY
+                read -r -p "[?] Create a cronjob/systemd-service which updates /etc/hosts with new adservers? [systemd/cronjob/N] " REPLY
                 case "${REPLY}" in
                     [Cc][Rr][Oo][Nn][Jj][Oo][Bb] | [Cr][Rr][Oo][Nn][Tt][Aa][Bb] | [Cc][Rr][Oo][Nn] | [Cc] ) # CRONJOB, CRONTAB, CRON, C
+                        read -r -p "[?] How often should the cronjob run? [weekly/DAILY/hourly/reboot] " FREQUENCY
+                        # set daily as default
+                        line="1 12 * * * root ${SCRIPT_DIR}/adaway-linux.sh"
+                        # check input
+                        case "${FREQUENCY}" in 
+                            [Ww][Ee][Ee][Kk][Ll][Yy] | [Ww] )
+                                line="1 12 */5 * * root ${SCRIPT_DIR}/adaway-linux.sh"
+                                ;;
+                            [Hh][Oo][Uu][Rr][Ll][Yy] | [Hh] )
+                                line="1 * * * * root ${SCRIPT_DIR}/adaway-linux.sh"
+                                ;;
+                            [Rr][Ee][Bb][Oo][Oo][Tt] | [Rr] )
+                                line="@reboot root ${SCRIPT_DIR}/adaway-linux.sh"
+                                echo "[i] Keep in mind that you need a working internet connection on every startup for this option."
+                                echo "    This may be the case when you are on wire but often isn't the case when you are using wireless networks."
+                                ;;
+                        esac
                         echo "[i] Creating cronjob..."
-                        line="1 12 */5 * * ${SCRIPT_DIR}/adaway-linux.sh"
-                        (crontab -u root -l; echo "${line}" ) | crontab -u root -
+                        echo "${line}" > ${CRONJOB_FILE}
+                        # make sure permissions are right
+                        chmod u=rw,g=r,o=r ${CRONJOB_FILE}
+                        chown root:root ${CRONJOB_FILE}
+                        # restart the cron service
+                        service cron restart 
```

Comment:
Do we really need to restart a service? I tested on fedora and there is neither a cron service nor you have to restart anything. Additionally using service is deprecated on most distributions that use systemd.